recipes/py2app/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "py2app" %}
+{% set version = "0.21" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 4516dfb01a0bf560ca4918e1e0e38eb4fcc76c9c2f527456e022310a80d52dc8
+
+build:
+  skip: true  # [not osx]
+  number: 0
+  entry_points:
+    - py2applet = py2app.script_py2applet:main
+  missing_dso_whitelist:
+    - '*/libgcc_s.1.dylib'
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
```

Comment:
please remove setuptools, since pip is a dep

Reply:
Fixed!